### PR TITLE
DataSeeding/EVentAdress

### PR DESCRIPTION
**Pull request by sushmakoosukuntla**
DataSeeding/EVentAdress

### CODE REVIEW

File: EventCatalogAPI/Data/EventContext.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public EventContext(DbContextOptions options) : base(options)
 
         }
 
-        public DbSet<EventAddress>  Addresses { get; set; }
```
Comment (misurac) [inline EventCatalogAPI/Data/EventContext.cs]: I think the table name should remain as Addresses, since it will be a table with multiple addresses

Comment (sushmakoosukuntla) [reply EventCatalogAPI/Data/EventContext.cs]: Ok, I can change that 👍 Thanks

Comment (misurac) [reply EventCatalogAPI/Data/EventContext.cs]: Great!

File: EventCatalogAPI/Data/EventContext.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public EventContext(DbContextOptions options) : base(options)
 
         }
 
-        public DbSet<EventAddress>  Address { get; set; }
+        public DbSet<EventAddress>  Addresses { get; set; }
```
Comment (misurac) [inline EventCatalogAPI/Data/EventContext.cs]: Looks awesome

**Review by SemraChernet — COMMENTED** [review]
Looks good to me. The only minor change I would make is way to Way.

**Pull request merged**